pkg/cmd/pr/status/http.go

```diff
@@ -188,6 +186,17 @@ func pullRequestStatus(httpClient *http.Client, repo ghrepo.Interface, options r
 	return &payload, nil
 }
 
+func getCurrentUsername(username string, hostname string, apiClient *api.Client) (string, error) {
```

Comment:
I don't this needs to happen as part of this PR but there is this thing I came across called a `MeReplacer` which is a struct that basically does this and already has its own tests. Might be something we could use here.

Reply:
I'll make an issue for it 👍